librz/core/disasm.c

```diff
@@ -1801,32 +1801,43 @@ static void ds_pre_xrefs(RzDisasmState *ds, bool no_fcnlines) {
 	ds->line_col = tmp_col;
 }
 
-static void ds_show_functions_stack_argvar(RzDisasmState *ds, RzAnalysisFunction *fcn, RzAnalysisVar *var) {
-	rz_return_if_fail(var->storage.type == RZ_ANALYSIS_VAR_STORAGE_STACK);
-	st64 off = var->storage.stack_off;
-	char sign = off >= 0 ? '+' : '-';
+static void ds_show_function_var(RzDisasmState *ds, RzAnalysisFunction *fcn, RzAnalysisVar *var) {
 	const char *pfx = rz_analysis_var_is_arg(var) ? "arg" : "var";
 	char *constr = rz_analysis_var_get_constraints_readable(var);
 	char *vartype = rz_type_as_string(ds->core->analysis->typedb, var->type);
-	rz_cons_printf("%s%s %s%s%s%s %s%s%s%s@ stack %c 0x%" PFMT64x,
+	rz_cons_printf("%s%s %s%s%s%s %s%s%s%s@ ",
```

Comment:
I think instead of this magic, we can update `type_as_pretty_string()` and `rz_type_as_pretty_string()` in `librz/type/type.c` (they already have various options to control the type output), but it is definitely out of the scope of this PR. cc @wargio